go/common/sgx/pcs/cache.go

```diff
@@ -50,18 +59,57 @@ type tcbBundleCache struct {
 	LastUpdate     time.Time  `json:"last_update"`
 }
 
+type tcbEvaluationDataNumbersCache struct {
+	Numbers    []uint32  `json:"numbers"`
+	LastUpdate time.Time `json:"last_update"`
+}
+
 type tcbCache struct {
 	serviceStore *persistent.ServiceStore
 	logger       *logging.Logger
 	now          func() time.Time
 }
 
-func (tc *tcbCache) check(fmspc []byte) (*TCBBundle, bool) {
+func (tc *tcbCache) checkEvaluationDataNumbers(teeType TeeType) ([]uint32, bool) {
+	var stored tcbEvaluationDataNumbersCache
+	switch err := tc.serviceStore.GetCBOR(tcbEvaluationDataNumbersCacheKey(teeType), &stored); err {
+	case nil:
+		// No error, continues below.
+	case persistent.ErrNotFound:
+		// Not cached yet. Not an error, but needs refresh.
+		return nil, true
+	default:
+		// Can't get it... an error, but we can still try downloading it.
+		tc.logger.Warn("error checking common store for cached TCB evaluation data numbers",
+			"err", err,
+		)
+		return nil, true
+	}
+
+	now := tc.now()
+	delta := now.Sub(stored.LastUpdate)
+	refresh := delta > tcbCacheSlowRefreshInterval
+	return stored.Numbers, refresh
+}
+
+func (tc *tcbCache) cacheEvaluationDataNumbers(teeType TeeType, numbers []uint32) {
+	cached := tcbEvaluationDataNumbersCache{
+		Numbers:    numbers,
+		LastUpdate: tc.now(),
+	}
+	if err := tc.serviceStore.PutCBOR(tcbEvaluationDataNumbersCacheKey(teeType), cached); err != nil {
+		tc.logger.Error("could not store new TCB evaluation data numbers to cache, ignoring",
```

Comment:
This should be logged by the caller.

Reply:
This is consistent with other code in the file. I don't think the caller can do much here.